views/index.erb

```diff
@@ -1,10 +1,13 @@
 <div class="welcome_message">
-  <h2>Welcome to the Compliance Viewer</h2>
+  <h2>Welcome to the Compliance Viewer!</h2>
   <p>
-    Compliance Viewer aims to make the ATO process and regulatory compliance more tangible, actionable, and transparent for project teams. It does this by automating processes that can be automated and assisting with the parts that require a human touch. The current focus of the site is on automated scanning of web applications, but the scope will expand as the project evolves.
+    The Compliance Viewer aims to make the Authority to Operate (ATO) process and regulatory compliance more tangible, actionable, and transparent for your project team.
```

Comment:
Maybe change "project team" to "everyone involved", as CV is meant to help make the process tangible for all those involved throughout the process.